samples_project/Assets/SampleViewer/Samples/FeatureLayer/readme.md

```diff
@@ -4,27 +4,44 @@ Query objects from a feature layer and show it on the map.
 
 ![Feature layer](FeatureLayer.png)
 
-## How to use the sample
+## How to Setup (Feature Layer Level)
 
-1. Open the **FeatureLayer** scene.
-2. Click on the **ArcGISMap** game object in the **Hierarchy** window.
-3. Set your API key in the **Inspector** window.
-4. Click play and select a stadium from the lst.
+1. Open the **FeatureLayer** Scene.
+2. Click on the **ArcGISMap** Gameobject in the hierarchy window.
+3. Set your API key under Authentication section in the inspector window.
+4. Click play.
+
+
+## How to Setup (Sample Viewer)
+
+1. Click Play in Unity Editor.
+2. Input your API key under the **API Key Drop down**.
+3. Click the **Sample Drop Down** and select **Feature Layer**.
+
+## How to Use
+
+1. Already available is a Feature Layer of park trees, upon running the scene, these feature items will be spawned in at their default latitude and longitude and with all of the properties associated with that feature item. 
+2. Clicking on any of these objects under the **FeatureLayer** Gameobject in Unity to see it's latatude, longitude, and any properties recieved by the query.
+3. Under the outfields drop down, by default it will be set to "Get All Outfields". Clicking on any of the other outfields will deselect "Get All Outfields" and will only get the outfield selected. Multi-select is supported. Outfields are the properties associated with the individual features. **Note** you need to reprocess the feature layer query after changes outfields to get.
```

Comment:
```suggestion
3. Under the outfields dropdown, by default it will be set to "Get All Outfields". Selecting any of the other outfields will deselect "Get All Outfields". Multi-select is supported. Outfields are the properties associated with the individual features. **Note**: after modifying the outfields, it is necessary to reprocess the feature layer query for the changes to take effect.
```